msg/CellularStatus.msg

```diff
@@ -1,28 +1,34 @@
-uint64 timestamp	# time since system start (microseconds)
+uint64 timestamp # Time since system start [us]
 
-uint8 CELLULAR_STATUS_FLAG_UNKNOWN=0 # State unknown or not reportable
-uint8 CELLULAR_STATUS_FLAG_FAILED=1 # velocity setpoint
-uint8 CELLULAR_STATUS_FLAG_INITIALIZING=2 # Modem is being initialized
-uint8 CELLULAR_STATUS_FLAG_LOCKED=3	# Modem is locked
-uint8 CELLULAR_STATUS_FLAG_DISABLED=4	# Modem is not enabled and is powered down
-uint8 CELLULAR_STATUS_FLAG_DISABLING=5	# Modem is currently transitioning to the CELLULAR_STATUS_FLAG_DISABLED state
-uint8 CELLULAR_STATUS_FLAG_ENABLING=6 	# Modem is currently transitioning to the CELLULAR_STATUS_FLAG_ENABLED state
-uint8 CELLULAR_STATUS_FLAG_ENABLED=7  # Modem is enabled and powered on but not registered with a network provider and not available for data connections
-uint8 CELLULAR_STATUS_FLAG_SEARCHING=8  # Modem is searching for a network provider to register
-uint8 CELLULAR_STATUS_FLAG_REGISTERED=9  # Modem is registered with a network provider, and data connections and messaging may be available for use
-uint8 CELLULAR_STATUS_FLAG_DISCONNECTING=10  # Modem is disconnecting and deactivating the last active packet data bearer. This state will not be entered if more than one packet data bearer is active and one of the active bearers is deactivated
-uint8 CELLULAR_STATUS_FLAG_CONNECTING=11  # Modem is activating and connecting the first packet data bearer. Subsequent bearer activations when another bearer is already active do not cause this state to be entered
-uint8 CELLULAR_STATUS_FLAG_CONNECTED=12  # One or more packet data bearers is active and connected
+uint16 status # Status bitmap 1: Roaming is active
```

Comment:
I don't get the `1: Roaming is active` - isn't STATUS_FLAG_UNKNOWN set = bit 1?

But either way, can we please associate the enum by name. Sometimes enums are used by two things, and that is the only way for a parser to make the association

```suggestion
uint16 status # Status bitmap 1: Roaming is active [@enum=CELLULAR_STATUS_FLAG]
```